packages/backend-deployer/src/cdk_deployer_singleton_factory.ts

```diff
@@ -32,28 +36,32 @@ export type BackendDeployer = {
   destroy: (backendId: BackendIdentifier) => Promise<DestroyResult>;
 };
 
+const packageManagerControllerFactory = new PackageManagerControllerFactory();
+
 /**
  * Factory to create a backend deployer
  */
 export class BackendDeployerFactory {
   private static instance: BackendDeployer | undefined;
 
+  /**
+   * constructor - sets the packageManagerController
+   */
+  constructor(
+    private readonly packageManagerController = packageManagerControllerFactory.getPackageManagerController()
```

Comment:
No need for the default here. Just make it a required prop that is injected by the cli package

Reply:
Done